esx_service/volume_kv.py

```diff
@@ -42,24 +47,27 @@
 # The name of the VSAN policy applied to the VSAN volume. Invalid for Non-VSAN
 # volumes.
 VSAN_POLICY_NAME = 'vsan-policy-name'
+
+## Value for VSAN_POLICY_NAME
+DEFAULT_VSAN_POLICY = '[VSAN default]'
+
 # The size of the volume
 SIZE = 'size'
+DEFAULT_DISK_SIZE = "100mb"
 
 # The disk allocation format for vmdk
 DISK_ALLOCATION_FORMAT = 'diskformat'
-
 VALID_ALLOCATION_FORMATS = ["zeroedthick", "thin", "eagerzeroedthick"]
-
 DEFAULT_ALLOCATION_FORMAT = 'thin'
 
-## Values for given keys
-
-## Value for VSAN_POLICY_NAME
-DEFAULT_VSAN_POLICY = '[VSAN default]'
+# attach type. Default is independent.
+ATTACH_AS = 'attach_as'
 
-## Values for STATUS
-ATTACHED = 'attached'
-DETACHED = 'detached'
+# actual values to pass to VM reconfig events
+INDEPENDENT = 'independent_persistent'  # does not participate in vm snashot
+DEPENDENT   = 'persistent' # does participated in VM snapshot
```

Comment:
Should we make this "dependent_persistent"? Otherwise, it would give wrong impression that default is non-persistent.